test/shared/utils.test.ts

```diff
@@ -142,4 +143,15 @@ describe("utils", () => {
             }
         );
     });
+
+    describe("getShortcut", () => {
+        it("should replace `Mod` with appropriate modifier key for current platform", () => {
+            const shortcut = getShortcut("Mod-z");
+            expect(shortcut).toBe("Ctrl-z");
```

Comment:
Will this test fail if run on macOS? 🤔

Can we use Jest to mock `navigator.platform`? If so, we can then test that both macos and "other" scenarios work.

Reply:
Looks like `navigator.platform` is unset when these tests run. Since the test defaults to applying `Ctrl`, this test was passing. I've added [two tests](https://github.com/StackExchange/Stacks-Editor/pull/127/commits/9cb4159b7f14063c9991fb61ea7a09d58796c38c) that mock `navigator.platform`.

I was tempted to set `SunOS` as the non-Mac platform, but resisted and went with `Win32`.